static/update-manually.sh

```diff
@@ -23,8 +23,13 @@ tar -xf "./${ARCHIVE_NAME}" -C "./${FOLDER_NAME}"
 
 for BINARY_NAME in $BINARY_NAMES
 do
-  rm -f "./${BINARY_NAME}"
-  cp "./${FOLDER_NAME}/${BINARY_NAME}" "./${BINARY_NAME}"
+  if [ "$BINARY_NAME" == "hypervisor" ]; then
+    rm -f "./skywire-hypervisor"
+    cp "./${FOLDER_NAME}/${BINARY_NAME}" "./skywire-hypervisor"
+  else
+    rm -f "./${BINARY_NAME}"
+    cp "./${FOLDER_NAME}/${BINARY_NAME}" "./${BINARY_NAME}"
```

Comment:
The binary needs to be copied to /usr/bin/... as the systemd unit file expects the binary to be there. Consequently the previous binary needs to be deleted there as well. If the test pass and this change has been done, this should work.